plugins/aws-lambda/src/components/useProjectName.ts

```diff
@@ -23,10 +23,18 @@ export const useProjectName = (name: EntityCompoundName) => {
 
   const { value, loading, error } = useAsync(async () => {
     const entity = await catalogApi.getEntityByName(name);
-    return (
+    const lambdaNames =
       entity?.metadata.annotations?.['backstage.io/aws-lambda']?.split(',') ??
-      []
-    );
+      [];
+    const projectName =
+      entity?.metadata.annotations?.['github.com/project-slug'] ?? '';
+
+    return { lambdaNames, projectName };
   });
-  return { value, loading, error };
+  return {
+    value: value?.projectName,
```

Comment:
I think the name of this hook is a bit too specific. Given we're likely to add more perhaps something like useServiceEntityAnnotations ? Also, we should move this file out of components and into a src/hooks directory.